src/apps/companies/apps/edit-history/client/EditHistory.jsx

```diff
@@ -1,15 +1,17 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { H3 } from 'govuk-react'
+import { H3 } from '@govuk-react/heading'
+import pluralise from 'pluralise'
+
+import EditHistoryList from './EditHistoryList'
 
 function EditHistory ({ editHistory }) {
+  const numberOfChanges = `${editHistory.length} ${pluralise(editHistory.length, 'change')}`
```

Comment:
This could be simplified with:
```suggestion
  const numberOfChanges = pluralize('change', editHistory.length, true)
```

Reply:
👍

Reply:
This doesn't work.